### ISSUE TITLE
Kata Containers CI tests are failing

### ISSUE THREAD

**Issue by saschagrunert** [cri-o/cri-o#4648]
Right now the kata containers CI fails with the following error message:
```
10:20:46 Failed at 136: ./test_runner.sh ctr.bats
10:20:46 Makefile:228: recipe for target 'crio' failed
```
http://jenkins.katacontainers.io/job/kata-containers-crio-PR/16872/console

I'm opening this issue as a placeholder since I'm not sure if anyone is already looking at those.

**Comment by fidencio** [comment]
Today is a looong day, but I will talk try to sync with Salvador later Today / Tomorrow afternoon the worst case.

**Comment by fidencio** [comment]
8 days later, but here we are.

We had a mix of two issues happening:
1. We had an non deterministic issue related to using devicemapper, which is not exactly supported on CRI-O.  I've dropped the devicemapper usage / requirements on Kata Containers side, when testing against CRI-O.
2. https://github.com/cri-o/cri-o/commit/c0f293ad505fb79a41f2f6e81782b6b043d9a975 changed how runtimes are passed, affecting only what's tested against `master`, and to solve that I adapted the jenkins CI itself, on the Kata Containers side.
